modules/customize-certificates-api-add-named.adoc

```diff
@@ -78,3 +78,21 @@ spec:
         name: <secret>
 ...
 ----
+
+. Check that a new revision of the Kubernetes API server pods has rolled out. This will take several minutes.
++
+[source,terminal]
+----
```

Comment:
After Updated the API server, the kube-apiserver cluster operator `PROGRESSING` condition will change to `True` about one minute later, before this, `PROGRESSING` condition is still 'False`.

Reply:
I have added a statement that the operator may take a moment to start the rollout.